### PR TITLE
Run pytest

### PR DESCRIPTION
**Pull request by adeas31**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+            }
+          }
+          steps {
+            sh 'pytest testing'
+          }
         }
-
-      }
-      steps {
-        sh 'hostname'
```
Comment (sjoelund) [inline Jenkinsfile:12]: The hostname command is pretty useful when you need to figure out which damn machine ran the command :D

File: Jenkinsfile [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+              // Large image with full OpenModelica build dependencies; lacks omc and OMPython
+              label 'linux'
+              dir '.jenkins/python3'
+              additionalBuildArgs '--pull'
+            }
+          }
+          steps {
+            sh 'hostname'
+            sh 'HOME="$PWD" python3 setup.py install --user'
+            sh 'pytest testing'
```
Comment (sjoelund) [inline Jenkinsfile:18]: Don't you also need HOME="$PWD" for the pytest call?

Comment (adeas31) [reply Jenkinsfile:18]: Seems so

Comment (adeas31) [reply Jenkinsfile:18]: Wonder if pytest can find libs installed using `--user`.

**Pull request merged**